openapi3/schemas.py

```diff
@@ -134,6 +136,62 @@ def request_model(self, **kwargs):
         # TODO - this doesn't get nested schemas
         return self.get_request_type()(kwargs, self)
 
+    def _resolve_allOfs(self):
+        """
+        Handles merging properties for allOfs
+        """
+        if self._resolved_allOfs:
+            return
+
+        self._resolved_allOfs = True
+
+        if self.allOf:
+            for c in self.allOf:
+                if isinstance(c, Schema):
+                    self._merge(c)
+
+    def _merge(self, other):
+        """
+        Merges this Schema with the other, preferring the other schema's values
```

Comment:
This comment, if I'm reading correctly seems to be opposite of your PR description. The PR description suggests that the newest/latest schema would be preferential to the one(s) before it, but this docstring - to me at least - seems to infer that that the preceding schema would be preferred.